pkg/kubelet/dockertools/kube_docker_types.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dockertools
+
+import (
+	dockertypes "github.com/docker/engine-api/types"
+)
+
+// APIImage is the container information returned by ContainerList
+type APIImage dockertypes.Image
```

Comment:
I decided not to define our own types to make the interface cleaner. (See #23699)
Maybe just use `dockertypes.Image` instead. :)